File: ProAgil.Domain/Entities/Speaker.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using ProAgil.Domain.Identity;

namespace ProAgil.Domain.Entities
{
    public class Speaker
    {
        public int Id { get; set; }
        public string MiniCurriculum { get; set; }

        [ForeignKey("User")]
        public int UserId { get; set; }

        public User User { get; set; }
        public List<SocialNetWork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: src/Sds.Events.UI/src/app/models/Speaker.ts
import { SocialNetwork } from './SocialNetwork';
import { SpeakerEvent } from './SpeakerEvent';
export class Speaker {
    id: number;
    name: string;
    miniCurriculum: string;
    imageUrl: string;
    contactCell: string;
    contactEmail: string;
    socialNetworks: SocialNetwork[];
    speakerEvents: SpeakerEvent[];
}
